test/bdd/common_steps.go

```diff
@@ -623,6 +623,7 @@ func (d *CommonSteps) httpPostFileWithSignatureAndExpectedCode(url, path, pubKey
 }
 
 func (d *CommonSteps) httpPost(url, data, contentType string) error {
+	println(fmt.Sprintf("!!!DEREK in httpPost step method"))
```

Comment:
remove this msg

Reply:
Oops, didn't mean to commit this. I removed it.